app/src/main/java/org/wikipedia/activitytab/ActivityTabFragment.kt

```diff
@@ -301,6 +306,20 @@ class ActivityTabFragment : Fragment() {
                 }
             }
 
+            LaunchedEffect(timelineItems.loadState.refresh) {
+                if (!hasImpressionBeenSent &&
+                    timelineItems.loadState.refresh is LoadState.NotLoading) {
+                    val state = if (timelineItems.itemCount == 0) "empty" else "complete"
```

Comment:
Not sure, but could you explain why this only counts `timelineItems` for empty or complete here?